.github/workflows/cd-app.yml

```diff
@@ -32,5 +28,5 @@ jobs:
     name: Deploy
     uses: ./.github/workflows/deploy.yml
     with:
-      app_name: ${{ inputs.app_name || 'app' }}
+      app_name: "app"
```

Comment:
I may be wrong but my understanding is that we want this to be able to handle different folder names besides app?

Reply:
After discussion/further review, looks good